From 12b55e8654520aac7cb466e7c149a11a48cc3b0e Mon Sep 17 00:00:00 2001
From: Anthony King <anthonydking@slimroms.net>
Date: Sat, 31 Oct 2015 15:27:39 -0400
Subject: [PATCH] py3: post_process_props

Change-Id: Idf7ed5daa4eaa88f5421c798862ea6bb09593cdc
---

diff --git a/tools/post_process_props.py b/tools/post_process_props.py
index 5193d4b..6bb68c7 100755
--- a/tools/post_process_props.py
+++ b/tools/post_process_props.py
@@ -16,6 +16,13 @@
 
 import os, sys
 
+
+def iteritems(obj):
+  if hasattr(obj, 'iteritems'):
+    return obj.iteritems()
+  return obj.items()
+
+
 # Usage: post_process_props.py file.prop [blacklist_key, ...]
 # Blacklisted keys are removed from the property file, if present
 
@@ -63,7 +70,7 @@
   """
   check_pass = True
   buildprops = prop.to_dict()
-  for key, value in buildprops.iteritems():
+  for key, value in iteritems(buildprops):
     # Check build properties' length.
     if len(key) > PROP_NAME_MAX:
       check_pass = False
